npm/react/package.json

```diff
@@ -12,6 +12,9 @@
     "cy:run:debug": "node --inspect-brk ../../scripts/start.js --component-testing --run-project ${PWD}",
     "pretest": "yarn transpile",
     "test": "yarn cy:run",
+    "yarn": "yarn test-unit && yarn test-component",
```

Comment:
Is this step supposed to be called `yarn`? This doesn't seem correct.

Reply:
Good catch. I meant to replace line 14 with `"test": "yarn test-unit && yarn test-component",`